src/NCronJob/Scheduler/JobQueueManager.cs

```diff
@@ -65,37 +73,14 @@ public void RemoveQueue(string queueName)
     public SemaphoreSlim GetOrAddSemaphore(string queueName, int concurrencyLimit) =>
         semaphores.GetOrAdd(queueName, _ => new SemaphoreSlim(concurrencyLimit));
 
-    public CancellationTokenSource GetOrAddCancellationTokenSource(string queueName)
```

Comment:
Cancellation tokens aren't lazily created anymore. This provides us with a better control of their lifecyle.

Reply:
This isn't used anymre at all - so we can remove it.